ldapadmin/src/main/java/org/georchestra/ldapadmin/ws/backoffice/orgs/OrgsController.java

```diff
@@ -347,6 +369,58 @@ public void getOrgTypeVAlues(HttpServletResponse response) throws IOException{
     }
 
     /**
+     * Return configuration areas UI as json object. Configuration includes following values :
+     *
+     * - inital map center
+     * - initial map zoom
+     * - ows service to retrieve area geometry 'url'
+     * - attribute to use as label 'value'
+     * - attribute to use to group area 'group'
+     * - attribute to use as identifier 'key'
+     *
+     * Ex :
+     * {
+     *   "map" : { "center": [49.5468, 5.123486], "zoom": 8},
+     *   "areas" : { "url": "http://sdi.georchestra.org/geoserver/....;",
+     *               "key": "insee_code",
+     *               "value": "commune_name",
+     *               "group": "department_name"}
+     * }
+     */
+
+    @RequestMapping(value = PUBLIC_REQUEST_MAPPING + "/areaConfig.json", method = RequestMethod.GET)
+    public void getAreaConfig(HttpServletResponse response) throws IOException, JSONException {
+        JSONObject res = new JSONObject();
+        JSONObject map = new JSONObject();
+        // Parse center
+        String[] rawCenter = this.georConfig.getProperty("AreaMapCenter").split("\\s*,\\s*");
+        JSONArray center = new JSONArray();
+        center.put(Double.parseDouble(rawCenter[0]));
+        center.put(Double.parseDouble(rawCenter[1]));
+        map.put("center", center);
+        map.put("zoom", this.georConfig.getProperty("AreaMapZoom"));
+        res.put("map", map);
+        JSONObject areas = new JSONObject();
+        areas.put("url", this.georConfig.getProperty("AreasUrl"));
+        areas.put("key", this.georConfig.getProperty("AreasKey"));
+        areas.put("value", this.georConfig.getProperty("AreasValue"));
+        areas.put("group", this.georConfig.getProperty("AreasGroup"));
+        res.put("areas", areas);
+        ResponseUtil.buildResponse(response, res.toString(4), HttpServletResponse.SC_OK);
+    }
+
+
+    @ExceptionHandler(Exception.class)
+    @ResponseStatus(value = HttpStatus.INTERNAL_SERVER_ERROR)
+    @ResponseBody
+    public String handleException(Exception e, HttpServletResponse response) throws JSONException, IOException {
```

Comment:
For me, if you use `@responseBody` in your handler methods, a thrown exception won't return a 200 code status.